openrtb_ext/seat_non_bids.go

```diff
@@ -0,0 +1,78 @@
+package openrtb_ext
+
+import "github.com/prebid/openrtb/v19/openrtb2"
+
+// NonBidsWrapper contains the map of seat with list of nonBids
+type NonBidsWrapper struct {
```

Comment:
Why can't we previous name - `NonBids` and open it as public struct, instead?

Reply:
We have moved this struct to openrtb_ext package. This package already contains standard definitions of NonBid , NonBidExt, SeatNonBid etc.
Hence, `NonBids` name looks confusing.
Even Prebid is using Wrapper as standard example - requestWrapper, impWrapper.
Similar to that, I've renamed it to NonBidsWrapper